Description: Include all necessary files in the initrd for a graphical theme
Author: Simon Quigley <tsimonq2@ubuntu.com>
Origin: vendor
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/dracut/+bug/1596220
Last-Update: 2024-11-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/scripts/plymouth-populate-initrd.in
+++ b/scripts/plymouth-populate-initrd.in
@@ -470,10 +470,15 @@ inst ${PLYMOUTH_DATADIR}/plymouth/themes
 inst ${PLYMOUTH_PLUGIN_PATH}/text.so $INITRDDIR
 inst ${PLYMOUTH_DATADIR}/plymouth/themes/details/details.plymouth $INITRDDIR
 inst ${PLYMOUTH_PLUGIN_PATH}/details.so $INITRDDIR
+inst ${PLYMOUTH_PLUGIN_PATH}/label-pango.so $INITRDDIR
+inst ${PLYMOUTH_PLUGIN_PATH}/ubuntu-text.so $INITRDDIR
+inst_recur /usr/share/fonts/truetype/ubuntu $INITRDDIR
 inst ${PLYMOUTH_LOGO_FILE} $INITRDDIR
 inst @RELEASE_FILE@ $INITRDDIR
 inst ${PLYMOUTH_POLICYDIR}/plymouthd.defaults $INITRDDIR
 inst ${PLYMOUTH_CONFDIR}/plymouthd.conf $INITRDDIR
+inst /etc/alternatives/default.plymouth $INITRDDIR
+inst /etc/default/keyboard $INITRDDIR
 
 # Install xkb info
 if [ -d "${X11_DIRECTORY}" ]; then
